DogTales/src/player.hpp

```diff
@@ -1,10 +1,34 @@
 #pragma once
 #include <bave/app.hpp>
-#include <bave/graphics/shape.hpp>
+//#include <bave/graphics/shape.hpp>
+#include <bave/graphics/sprite.hpp>
+
+enum class Direction {
+	UP,
+	DOWN,
+	LEFT,
+	RIGHT
+};
+
+constexpr glm::vec2 PLAYER_SPEED { 500.0f, 500.0f };
```

Comment:
Move the constant inside `class Player` and make it `static`, then it doesn't need the `player_` prefix, as it's already in `Player::` scope.